recipe/build.sh

```diff
@@ -3,26 +3,21 @@ set -xeo pipefail
 
 rm -f subprojects/gtest.wrap
 EXTRA_MESON_ARGS=""
-if [[ ${variant} == "rpi" ]]; then
-  EXTRA_MESON_ARGS="${EXTRA_MESON_ARGS} -Dpycamera=enabled"
+EXTRA_MESON_ARGS="${EXTRA_MESON_ARGS} -Dgstreamer=enabled"
+EXTRA_MESON_ARGS="${EXTRA_MESON_ARGS} -Dtest=false"
+EXTRA_MESON_ARGS="${EXTRA_MESON_ARGS} -Dcam=enabled"
+EXTRA_MESON_ARGS="${EXTRA_MESON_ARGS} -Dpycamera=enabled"
+if [[ "${target_platform}" == linux-* ]]; then
+  EXTRA_MESON_ARGS="${EXTRA_MESON_ARGS} -Dv4l2=true"
+fi
+if [[ ${variant} == "rpi_fork" ]]; then
   EXTRA_MESON_ARGS="${EXTRA_MESON_ARGS} -Dpipelines=rpi/vc4,rpi/pisp"
   EXTRA_MESON_ARGS="${EXTRA_MESON_ARGS} -Dipas=rpi/vc4,rpi/pisp"
-  EXTRA_MESON_ARGS="${EXTRA_MESON_ARGS} -Dv4l2=true"
-  EXTRA_MESON_ARGS="${EXTRA_MESON_ARGS} -Dgstreamer=enabled"
-  EXTRA_MESON_ARGS="${EXTRA_MESON_ARGS} -Dtest=false"
-  EXTRA_MESON_ARGS="${EXTRA_MESON_ARGS} -Dlc-compliance=disabled"
-  EXTRA_MESON_ARGS="${EXTRA_MESON_ARGS} -Dcam=disabled"
   EXTRA_MESON_ARGS="${EXTRA_MESON_ARGS} -Dqcam=disabled"
-
-  # the latter two are needed to avoid build errors with GCC,
-  # see https://github.com/raspberrypi/libpisp/pull/43
-  EXTRA_MESON_ARGS="${EXTRA_MESON_ARGS} -Dc_args=-Wno-unused-parameter"
-  EXTRA_MESON_ARGS="${EXTRA_MESON_ARGS} -Dcpp_args=-Wno-unused-parameter"
```

Comment:
These are no longer necessary since we can use the conda-forge libpisp (see why we build things separately)

Reply:
yep, that totally makes sense! thanks!